it-tests/settings/NodeLabelSettings.test.ts

```diff
@@ -64,7 +64,7 @@ describe('User Settings', function () {
     it(`Check 'id' Node Label is used instead of default 'description'`, async function () {
         this.timeout(60_000);
 
-        const timer = await driver.findElement(locators.TimerComponent.timer).findElement(locators.TimerComponent.label);
+        const timer = await driver.findElement(By.css(`${locators.TimerComponent.timer} ${locators.TimerComponent.label}`));
```

Comment:
Semantically, this is not exactly the same as before.

Before, we were locating a node and from that point, we searched for another one within. Now, we're targeting the label that is inside of the node.

The issue is that for some reason, topology now creates invisible nodes, potentially as references inside of the groups, so it means the test library finds 2 nodes, instead of one, and the first one is empty, therefore, the location fails.